src/styles/header.scss

```diff
@@ -0,0 +1,67 @@
+@use './variables.scss' as *;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  text-transform: uppercase;
+  width: auto;
+  background-color: #fff;
```

Comment:
The background-color is hardcoded as '#fff'. Consider using a variable from your variables.scss file for consistency and easier theme management.